azure-proto-compute/VersionOverrides/AvailabilitySetsApiVersions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using azure_proto_core;
+
+namespace azure_proto_compute
+{
+    public class AvailabilitySetsApiVersions : ApiVersionsBase
+    {
+        public static readonly AvailabilitySetsApiVersions V2020_05_01 = new AvailabilitySetsApiVersions("2020-05-01");
+        public static readonly AvailabilitySetsApiVersions V2019_12_01 = new AvailabilitySetsApiVersions("2019-12-01");
+        public static readonly AvailabilitySetsApiVersions Default = V2020_05_01;
+
+        static AvailabilitySetsApiVersions()
+        {
+            _validValues.Add(V2020_05_01);
+            _validValues.Add(V2019_12_01);
```

Comment:
How is validValues used? I only see added to ever and never checked (contains called). Also, are not the valid values restricted by lines 7-8 since you cannot set AvailabilitySetsApiVersions any other way since the string operator is not implemented and the constructor is private?

Reply:
yeah you are right, after I removed the string -> ApiVersion implicit this is no longer required